Modules/_io/bufferedio.c

```diff
@@ -428,6 +428,18 @@ buffered_clear(buffered *self)
     return 0;
 }
 
+static PyObject *
+buffered_at_fork_reinit(buffered *self, PyObject *Py_UNUSED(ignored))
+{
+    int ret = _PyThread_at_fork_reinit(&self->lock);
+    self->owner = 0;
+    if (ret == 0) {
+        return Py_True;
+    } else {
```

Comment:
PEP 7 says to put `else` on a new line

Reply:
Thank you! This is my first time (trying to) contribute to python, so there's definitely a lot of stuff I need to learn re style and conventions. Will read PEP 7 and the reference counting documentation to get a better idea of the right way to do this. 

Thanks for the help!

Reply:
Welcome!